BitmarkRegistry/Helpers/BitmarkFileUtil.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  FileUtil.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 6/28/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+
+class BitmarkFileUtil {
```

Comment:
Empty class.

Reply:
I'm aware it; but then I'm implementing the encrypted function in this File in the next PR @@